README.md

```diff
@@ -180,9 +199,9 @@ The Elixir Problem Matchers in this repository are adapted from [here](https://g
 - `@v1.8`: the latest in the `1.8.z` series (this tag is movable),
 - `@v1.8.0`: release `1.8.0` (this tag is not movable).
 
-To prevent issues in CI (unless you're OK with potential incompatibility between versions - we
-make real a effort to not introduce those without bumping major) we suggest you to use `@vx.y.z`,
-whenever possible.
+We make a real effort to not introduce incompatibilities without changing the major
+version number. To be extra safe against changes causing issues in CI you can specify
+an exact version with `@vx.y.z`.
```

Comment:
I found this part a bit confusing so I took liberty to change it. I think this is more clear. Or maybe I misunderstood and specifying all three numbers is _always_ recommended?

Reply:
It's not _always_ recommended, but it's preferable if you want a more stable environment (and even so, the underlying Node.js packages change, so it's really not just up to us, unfortunately). Your wording is better than what was there before, in any case.